app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="login_logo_margin">48dp</dimen>
```

Comment:
Split the dimens in regions, kind of a good practice to became easy to search for something.